lib/podlet.js

```diff
@@ -259,28 +231,28 @@ const PodiumPodlet = class PodiumPodlet {
             target,
             schemas.manifest.uri,
             new Error(
-                `Value on argument variable "target", "${target}", is not valid`,
-            ),
+                `Value on argument variable "target", "${target}", is not valid`
+            )
         );
 
         joi.attempt(
             name,
             schemas.manifest.name,
             new Error(
-                `Value on argument variable "name", "${name}", is not valid`,
-            ),
+                `Value on argument variable "name", "${name}", is not valid`
+            )
         );
 
         if (Object.keys(this.proxyRoutes).length >= 4) {
             throw new Error(
-                `One can not define more than 4 proxy targets for each podlet`,
+                `One can not define more than 4 proxy targets for each podlet`
             );
         }
 
         this.proxyRoutes[name] = target;
 
         if (this.development) {
-            this._proxy.register(this);
+            this.httpProxy.register(this);
```

Comment:
I saw that one of the examples is referring to `this._proxy` is that correct or does it need to be updated?

Reply:
No. The example you refere to are more tests on implementing misc http frameworks. Will do those into proper modules and then delete the example.